src/esp/gfx/CubeMap.cpp

```diff
@@ -109,6 +109,12 @@ Mn::PixelFormat getPixelFormat(CubeMap::TextureType type) {
   CORRADE_INTERNAL_ASSERT_UNREACHABLE();
 }
 
+uint8_t textureIndex(CubeMap::TextureType type) {
+  uint8_t idx = static_cast<uint8_t>(type);
+  CORRADE_INTERNAL_ASSERT(idx < CubeMap::numTextureTypes);
+  return idx;
+}
```

Comment:
I thought this would wrap the `textures_` access, to avoid accidentally accessing the member directly without a range check. That will also be shorter -- instead of

```cpp
textures_[textureIndex(type)]
```

everywhere, you'd do just

```cpp
getTexture(type)
```

for example. For even more "security" you could rename `textures_` to `textures_DO_NOT_TOUCH_` :D